api/repo/update.go

```diff
@@ -212,6 +222,12 @@ func UpdateRepo(c *gin.Context) {
 		r.SetAllowPull(true)
 		r.SetAllowPush(true)
 	}
+	// -- END DEPRECATED SECTION --
+
+	// set default events if no events are enabled
+	if r.GetAllowEvents().ToDatabase() == 0 {
```

Comment:
nit: i know this is in types, but i kind of wish `ToDatabase` was something like `ToBitmask` or something that signals why it's returning an int here.